### PR TITLE
fix(config): add parameters and correct default values for Fibaro Smart Module FGS214 and Double Smart Module FGS224

### PR DESCRIPTION
**Pull request by cyrilst**
Added missing parameters (35, 40, 41, 159 and 161).
Changed incorrect default values (for parameters 20 and 21).
Improved descriptions according to Fibaro documentation.
Changed On and Off case for consistency.

### CODE REVIEW

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Thanks, but your changes do not conform with our config file [style guide](https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide?id=configuration-parameters). Please read through the section on config parameters and apply it.

Note that we already have some predefined templates, e.g. for the [Scenes Sent](https://github.com/zwave-js/node-zwave-js/blob/93aa398d8df97f35e17cfa0d000ee785a2015cd6/packages/config/config/devices/0x010f/templates/fibaro_template.json#L67-L86), which should be used here. You can take a look at files for other similar Fibaro devices to see how they are used.

**Comment by cyrilst** [comment]
Thanks for the pointers @AlCalzone 
Just updated the patch, don't hesitate to tell me if there is anything to improve.
When this patch will be ok, I'll update the FGS224 the same way.

File: packages/config/config/devices/0x010f/fgs214.json [hunk 1]
```diff
@@ -35,121 +35,297 @@
...
 					"value": 0
 				},
 				{
-					"label": "Toggle switch (contact closed - On, contact opened - OFF)",
+					"label": "Toggle switch synchronized",
 					"value": 1
 				},
 				{
-					"label": "Toggle switch (device changes status when switch changes status)",
+					"label": "Toggle switch with memory",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x010f/fgs214.json]: I'm not 100% happy with these, the original is a bit verbose, but the shortened version is not self-explanatory. What about something along these lines?

value 1 -> `"Toggle switch (synchronize output)"`
value 2 -> `"Toggle switch (toggle output)"`

File: packages/config/config/devices/0x010f/fgs214.json [hunk 2]
```diff
@@ -35,121 +35,297 @@
...
```
Comment (AlCalzone) [inline packages/config/config/devices/0x010f/fgs214.json]: dito

File: packages/config/config/devices/0x010f/fgs214.json [hunk 3]
```diff
@@ -35,121 +35,297 @@
...
 					"value": 0
 				},
 				{
-					"label": "Restore remembered state of relays after restoring power",
+					"label": "Previous state",
 					"value": 1
 				},
 				{
-					"label": "Restore remembered state of relays after restoring power, but for toggle switches (parameter 20/21 set to 1) set the same state as the current state of the switches",
+					"label": "Same as switches",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x010f/fgs214.json]: I think this needs to mention the behavior when toggle switch (synchronized) is selected. Also "same as switches" is only true when param 20/21 is not set to 1.

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Thanks, much better. Just a few nits.

File: packages/config/config/devices/0x010f/fgs214.json (hunk 1 above)
Comment (cyrilst) [reply packages/config/config/devices/0x010f/fgs214.json]: Indeed, that's way better.

File: packages/config/config/devices/0x010f/fgs214.json (hunk 3 above)
Comment (cyrilst) [reply packages/config/config/devices/0x010f/fgs214.json]: Something like "Toggle switch (synchronize output): same as switches – other switch types: previous state"? I'm not convinced it's clear enough.

Comment (AlCalzone) [reply packages/config/config/devices/0x010f/fgs214.json]: Lets do "Toggle switch (synchronize output): current switch state / Other switch types: previous state"

Comment (cyrilst) [reply packages/config/config/devices/0x010f/fgs214.json]: Ok, I just pushed the updated version.

**Comment by AlCalzone** [comment]
LGTM. Do you want to do the FGS224 as part of this PR too?

**Review by AlCalzone — DISMISSED** [review]

**Comment by cyrilst** [comment]
Why not, the file will really look similar. Should I do a new commit for FGS224 in the same PR?

**Comment by AlCalzone** [comment]
Sure, go for it.

**Comment by cyrilst** [comment]
Done. Maybe you should have a look at the last parameter (164). I changed the wording according to the style guide, but I'm not sure if it's self-explanatory.

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**